docs/scos/dev/the-docker-sdk/202204.0/the-docker-sdk.md

```diff
@@ -15,12 +15,219 @@ related:
+
+The `docker/sdk down` command executes `docker-compose down`, which stops and removes containers and networks.
+
+The `docker/sdk prune` command does the following:
+- Executes the `docker-compose down -v` command.
+- Executes the `docker prune` command for images, volumes, system, and builder.
+
+The `docker/sdk reset`  command does the following:
+- executes `docker-compose down -v`
+- executes `docker/sdk up --build --assets --data --jobs`
+
+The `docker/sdk clean-data` command stops and removes the following containers and volumes with  `docker-compose down -v`:
+- logs
+- `/data` folder for `zed`, `yves`, `glue`
+- broker data: `/var/lib/rabbitmq`
+- session and key-value data: `/data`
+- search data: `/usr/share/elasticsearch/data`
+- scheduler data
+
+The `docker/sdk trouble` command does the following:
+* Executes `docker-compose down`.
+* Cleans sync volumes for `docker-sync`.
+
+The `docker/sdk build` command builds the following data:
+- Images: executes the `docker build` command for app and Nginx images; builds codebase and assets.
+- Codebase: executes `composer install`, `install -s build`, `build-{mode}`.
+- Assets: executes `install -s build-static`, `build-static-{mode}`.
+
+The `docker/sdk pull` command executes `docker-compose pull`, which pulls all the external images, based on `docker-compose.yml`.
+
+The `docker/sdk demo` command does the following:
+ - Executes the `docker-compose up -d` command.
+ - Runs the demo data installation process for each region.
+
+The `docker/sdk export images {tag}` command is only for the baked mode. It does the following:
+ - Builds and tags docker application images.
+ - Builds assets builder docker image based on assetsBuilder and Cli image). Runs `vendor/bin-install -s build-static build-static-{mode}`.
+ - Builds and tags frontend images.
+ - Prints information about the built images.
+
+## Development environment
+The following schema illustrates the development environment within the Docker SDK:
+![development-environment](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/dev/the-docker-sdk/the-docker-sdk/development-environment.png)
+
+In the development environment, Gateway (Nginx) does the following:
+
+* Accepts all public HTTP requests.
+* Handles SSL connections.
+* Serves all non-application services.
+* Proxies application calls to Frontend (Nginx).
+
+Fontend (Nginx), translates HTTP to FastCGI proxy for Glue, Yves, Zed, and serves assets.
+
+## Docker/sdk debug mode
+
+The following schema illustrates the Docker/sdk debug mode:
+
+![docker-sdk-debug-mode](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/dev/the-docker-sdk/the-docker-sdk/docker-sdk-debug-mode.png)
+
+## Docker/sdk in SCCOS
+The following schema illustrates how Docker/sdk is used in the Spryker Cloud Commerce OS (SCCOS):
+![docker-sdk-in-sccos](https://s3.console.aws.amazon.com/s3/object/spryker?region=eu-central-1&prefix=docs/scos/dev/the-docker-sdk/the-docker-sdk/docker-sdk-in-sccos.png)
```

Comment:
broken image? I can't see it in preview mode

Reply:
I've fixed the link, you should see the image now